src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+// Actions
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+// Reducer
+const reducer = (state = [], action = {}) => {
+  let newState;
+  switch (action.type) {
+    case ADD_BOOK:
+      newState = [...state];
+      return newState.push(action.payload);
+
+    case REMOVE_BOOK:
+      newState = state.filter((book) => book.id !== action.payload.id);
+      return newState.push(action.payload);
+
+    default: return state;
+  }
```

Comment:
- [ ] at this stage you should be able to display books and create books using your redux code by dispatching the book when creating and removing with an id of the book right now your application it's now working as expected so make sure to add the code to implement that